jenkins-plugin/pom.xml

```
@@ -424,7 +424,7 @@
       <dependency>
         <groupId>com.google.code.findbugs</groupId>
         <artifactId>annotations</artifactId>
-        <version>3.0.0</version>
+        <version>3.0.1u2</version>
```

```suggestion
        <version>3.0.1</version>
```

Let's stick to the latest "stable"